.github/workflows/build-rd-cpp.yml

```diff
@@ -0,0 +1,204 @@
+name: Build RD CPP
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  NINJA_VERSION: 1.9.0
+  BUILD_TYPE: Release
+  VERSION: 201.1
+  # Need to change these values to JetBrains account
+  # And set `RD_CPP_BINTRAY_KEY` for RD repo
+  BINTRAY_USER: alexander-pirogov
```

Comment:
It looks like forgotten debug value.

Reply:
I've deliberately left these values untill I'll have an official creds for JetBrains bintray account.

Reply:
I think we need to wait for it before merging.